src/test/java/SetTest.java

```diff
@@ -0,0 +1,41 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+    @Test
+    @DisplayName("Set.size() 에 대한 학습 테스트")
+    void sizeMethodTest() {
+        assertThat(numbers.size()).isEqualTo(3);
```

Comment:
```suggestion
        assertThat(numbers).hasSize(3);
```

collection의 경우 위와 같이 구현도 가능함